include/depends.mk

```diff
@@ -13,7 +13,7 @@
 
 DEP_FINDPARAMS := -x "*/.svn*" -x ".*" -x "*:*" -x "*\!*" -x "* *" -x "*\\\#*" -x "*/.*_check" -x "*/.*.swp" -x "*/.pkgdir*"
 
-find_md5=find $(wildcard $(1)) -type f $(patsubst -x,-and -not -path,$(DEP_FINDPARAMS) $(2)) | mkhash md5
+find_md5=find $(wildcard $(1)) -type f $(patsubst -x,-and -not -path,$(DEP_FINDPARAMS) $(2)) -printf "%p%TY%Tm%Td%TH%TM%TS%Tz\n" | sort | mkhash md5
```

Comment:
Wouldn't it be easier to use e.g. `%p%TF%TZ`?

Reply:
Hi @aparcar - Thanks for looking at this change.

I've updated the change to use the shorter form arguments and retested.